kubernetes/ftest-kubernetes/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://jboss.org/schema/arquillian"
+    xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <extension qualifier="kubernetes">
```

Comment:
I think that if no attribute is used, you don't need to add it, but leave as is, it doesn't matter, just to let you know :)